response_operations_ui/templates/conversation-view/cv-reopen-form.html

```diff
@@ -2,7 +2,7 @@
 {% set isMessageEditPermission = hasPermission('messages.edit') %}
 
 {% if isMessageEditPermission %}
-  <form action="{{ url_for('messages_bp.view_conversation', thread_id=messages[0].thread_id, ru_ref_filter=ru_ref_filter, business_id_filter=business_id_filter) }}" method="post">
+  <form action="{{ url_for('messages_bp.view_conversation', thread_id=messages[0].thread_id, conversation_tab=conversation_tab, ru_ref_filter=ru_ref_filter, business_id_filter=business_id_filter) }}" method="post">
```

Comment:
We weren't passing through the conversation_tab to allow the request to redirect to the closed tab, this has been fixed and the test updated.